README.md

```diff
@@ -188,13 +188,13 @@ You can find more information in the [vscode-theme-gotham][vscode-theme-repo] re
 
 ### Atom
 
-A Gotham-inspired syntax theme, with special features, is available for Atom: [atom-ubik-syntax](atom-ubik-syntax).
+A Gotham-inspired syntax theme, with special features, is available for Atom: [atom-ubik-syntax][atom-ubik-syntax].
 
 **Installation:** `apm install ubik-syntax`
 
 ### Spyder
 
-A Gotham-inspired syntax theme is available for Spyder: [spyder-ubik-syntax](spyder-ubik-syntax)
+A Gotham-inspired syntax theme is available for Spyder: [spyder-ubik-syntax][spyder-ubik-syntax]
```

Comment:
You should not be sorry, it happens :) I just pushed the changes you made in this PR directly to master (5eb61b3584c5319a21db74a8d89031eab8ac785e).